README.md

```diff
@@ -1,246 +1,223 @@
+After you modify your deployable architecture in projects, you can share it with others through a private IBM Cloud catalog. To share your deployable architecture, follow the steps in [Sharing your deployable architecture to your enterprise](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-share-custom).
 
-# Undeploying/Deleting the Stack, and all associated Infrastructure Resources
+### Customizing your application
 
-## Cleanup the configuration
+You can use the code of this sample automation as a guide to customize the sample app to meet your requirements. The code is available at [https://github.com/terraform-ibm-modules/terraform-ibm-rag-sample-da](https://github.com/terraform-ibm-modules/terraform-ibm-rag-sample-da).
 
-> This step is optional if you are planning to fully destroy all Watson resources. The artifacts created by the application will be deleted as part of undeploying the Watson resources.
+To use your own app, remove the `Workload - Sample RAG App Configuration` member configuration from the stack. This member configuration is specific to the default sample app.
 
-Follow the steps outlined in the [cleanup.md file](https://github.com/IBM/gen-ai-rag-watsonx-sample-application/blob/main/artifacts/artifacts-cleanup.md) file to remove the configuration specific to the sample app.
+## Undeploying the stack and infrastructure
 
-## Undeploying Infrastructure
 
-To undeploy the infrastructure created by the automation, complete the following steps:
+1.  Clean up the configuration
 
-### 1. Delete Resources Created by the CI toolchain
+    This step is optional if you plan to destroy all Watson resources. The artifacts that are created by the application are deleted as part of undeploying the Watson resources.
 
-Those resources are not destroyed automatically as part of undeploying the stack in Project:
-- **Code Engine Project**: Delete the code engine project created for the sample application.
-- **Container Registry Namespace**: Delete the container registry namespace created by the CI tookchain.
+    Follow the steps outlined in the [cleanup.md](https://github.com/IBM/gen-ai-rag-watsonx-sample-application/blob/main/artifacts/artifacts-cleanup.md) file to remove the configuration for the sample app.
 
+1.  Delete resources created by the CI toolchain
 
-### 2. Undeploy Configurations in the Project
+    The following resources, which are created by the toolchain, are not destroyed as part of undeploying the stack in Project.
 
-Select "Undeploy" option in the menu associated with the stack in the project.
-![undeploy](./images/17-undeploy.png)
+    - Code Engine Project
+        - Delete the code engine project created for the sample application.
```

Comment:
You no longer need to manually do this - the code engine projects are now managed by terraform in the latest version